common/src/main/java/com/wynntils/models/raid/RaidModel.java

```diff
@@ -348,6 +388,17 @@ private List<Long> getAllRoomTimes() {
         return allRoomTimes;
     }
 
+    private List<Long> getAllRoomDamages() {
+        List<Long> allRoomDamages = new ArrayList<>();
+
+        allRoomDamages.add(getRoomDamage(RaidRoomType.CHALLENGE_1));
+        allRoomDamages.add(getRoomDamage(RaidRoomType.CHALLENGE_2));
+        allRoomDamages.add(getRoomDamage(RaidRoomType.CHALLENGE_3));
+        allRoomDamages.add(getRoomDamage(RaidRoomType.BOSS_FIGHT));
+
+        return allRoomDamages;
+    }
```

Comment:
This is called by functions, right? If so, it's likely wise to cache this list, and return an immutable view `Collections#unmodifiableList` of the cached version.

Reply:
No this is just to get all the damages for the raid completed event, the timers do the same thing